dashboard/checkit/outcome.py

```diff
@@ -50,7 +50,8 @@ def to_dict(self,regenerate=False):
 
     def preview_exercises(self):
         preview_json = os.path.join(self.build_path(),"preview.json")
-        sage(self,preview_json,preview=True,images=True)
+        # hardcode: previews generate 10 seeds
+        sage(self,preview_json,amount=10,preview=True,images=True)
```

Comment:
`amount=10` will be ignored by the sage script (see next comment)

Reply:
Ok, yep, this works. (I was worried that the signature wouldn't match but it's ok because we handle a default amount in wrapper/__init__.py.)